### ISSUE TITLE
G3 / G7 - search button accessibility issues

### ISSUE THREAD

**Issue by jszasz** [brharp/hjckrrh#73]
![g3](https://cloud.githubusercontent.com/assets/12450523/8362695/662ea242-1b48-11e5-8f84-5c190a8b0f36.PNG)

G3 - local

![g7](https://cloud.githubusercontent.com/assets/12450523/8362698/6856c824-1b48-11e5-8e05-0c30b8a7b9c8.PNG)

G7 - Global

both searches look weird with no styles, and sounds weird on screen reader. G3 has two "buttons" and G7 doesn't tell you what the button does, just says "button".

**Comment by brharp** [comment]
Added descriptive text to the global search button (G3). Both buttons on G7 also have text. There are still 2 buttons on G7 if you disable CSS, but the second button should be invisible to all (including screen readers) if CSS is enabled.
